src/sass/details-group/details-group.scss

```diff
@@ -0,0 +1,59 @@
+@import "../variables/variables";
+@import "../mixins/private/token-mixins";
+
+ul.details-group {
```

Comment:
If there's no spec for `:hover` and `:active`, it might be best to unset those.

Specifically, for `:active`, this looks kind of bad; the spacing between the `summary` and `details` looks broken.

<kbd><img width="1052" alt="image" src="https://github.com/user-attachments/assets/ffb73604-d89c-45a0-b81f-b1165ea5959b"></kbd>

Reply:
By adding the `details__content` it fixes with additional spacing.